test-project/psp22_contract/Cargo.toml

```diff
@@ -5,29 +5,26 @@ authors = ["Brushfam <green@727.ventures>"]
 edition = "2021"
 
 [dependencies]
-ink = { version = "4.1.0", default-features = false}
+ink = { version = "4.2.1", default-features = false }
```

Comment:
```suggestion
ink = { version = "=4.2.1", default-features = false }
```

Reply:
I'm against this, for example this makes dependabot unable to automatically create PRs bumping these, and generates some other issues I bumped into in the past.

Reply:
It might use ink `4.99.99` then when building the image.

Reply:
Firstly, this is just the contract used for the test. Secondly, only the last digit of the version is unlocked (except for <1.0.0, not the case here), so at most it can use `4.2.99`.

Reply:
Right, MB, I had it backwards.